packages/rx_bloc_cli/lib/src/models/generator_arguments_provider.dart

```diff
@@ -98,11 +105,14 @@ class GeneratorArgumentsProvider {
       _logger.warn('Pin code enabled, due to MFA feature requirement');
       pinCodeEnabled = true;
     }
-    if ((otpEnabled || pinCodeEnabled || onboardingEnabled) &&
+    if ((otpEnabled ||
+            pinCodeEnabled ||
+            onboardingEnabled ||
+            forgottenPassword) &&
         !(loginEnabled || socialLoginsEnabled)) {
       // Modify feature flag or throw exception
-      _logger
-          .warn('Login enabled, due to OTP/PIN/Onboarding feature requirement');
+      _logger.warn(
+          'Login enabled, due to OTP/PIN/Onboarding/Forgotten Pass feature requirement');
```

Comment:
Can we specify the `Forgotten password` in its full format so we are consistent with the outputs?